Project-II-Calculator/calculator/src/App.js

```diff
@@ -1,23 +1,82 @@
 import React from 'react';
 import './App.css';
+import CalculatorDisplay from './components/DisplayComponents/CalculatorDisplay';
+import ButtonContainer from './components/ButtonComponents/ButtonContainer';
+import ActionButton from './components/ButtonComponents/ActionButton';
+import NumberButton from './components/ButtonComponents/NumberButton';
 
-const App = () => {
-  return (
-    <div>
-      <h3>Welcome to React Calculator</h3>
-      <p>
-        We have given you a starter project. You'll want to build out your
-        components in their respective files, remove this code and replace it
-        with the proper components.
-      </p>
-      <p>
-        <strong>
-          Don't forget to `default export` your components and import them here
-          inside of this file in order to make them work.
-        </strong>
-      </p>
-    </div>
-  );
-};
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = { display: '0', total: 0, op: 'number', rh : 0 };
+
+  }
+  
+  newNumber( keyPressed) {
```

Comment:
good job keeping all your logic here. 

I gotta say when I saw all these if else statements I thought you were writing and AI app for a second lol (joking) but I'm glad you knocked out the stretch and added such great functionality to your app.